articles/aks/operator-best-practices-scheduler.md

```diff
@@ -121,7 +121,7 @@ The [kube-advisor][kube-advisor] tool is an associated AKS open source project t
 
 The kube-advisor tool can report on resource request and limits missing in PodSpecs for Windows applications as well as Linux applications, but the kube-advisor tool itself must be scheduled on a Linux pod. You can schedule a pod to run on a node pool with a specific OS using a [node selector][k8s-node-selector] in the pod's configuration.
 
-In an AKS cluster that hosts multiple development teams and applications, it can be hard to track pods without these resource requests and limits set. As a best practice, regularly run `kube-advisor` on your AKS clusters, especially if you don't assign resource quotas to namespaces.
+In an AKS cluster that hosts multiple development teams and applications, it can be hard to track pods without these resource requests and limits set. As a best practice, regularly run `kube-advisor` on your AKS clusters, especially if you assign resource quotas to namespaces.
```

Comment:
I would rewrite this block to state that it is very highly recommended to implement resource limits - unless this is heavily captured elsewhere in the doc. Update second sentence to state that we recommend to run kube-advisor to discover pods missing resource and request limits.

That should avoid the confusion on the whether or not you assign resource limits because you always should.